fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/GiveGiftsToHeroTaskAccessor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import java.util.Map;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.entity.ai.brain.task.GiveGiftsToHeroTask;
+import net.minecraft.util.Identifier;
+import net.minecraft.village.VillagerProfession;
+
+@Mixin(GiveGiftsToHeroTask.class)
+public interface GiveGiftsToHeroTaskAccessor {
+	@Accessor("GIFTS")
+	static Map<VillagerProfession, Identifier> getGifts() {
```

Comment:
This is a bad name. If some other mod does the same thing they will likely choose the same (obvious) name.
You then have a conflict because of duplicate method definitions in the same class.

I believe the standard (though not always followed) is to append "fabric_", e.g.
https://github.com/FabricMC/fabric/blob/1d2d03ecd5a4508d14d4bf0647324afef1b8d2bd/fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/mixin/transfer/BucketItemAccessor.java#L28

Reply:
I have fixed this for all the mixins added in this PR

Reply:
Huh? Why would this be a problem? The accessor is static, and won't Mixin merge accessors to the same field anyway?